### ISSUE TITLE
ICD10 code R82.4 maps to two concepts in Measurement domain

### ISSUE THREAD

**Issue by schuemie** [OHDSI/Vocabulary-v5.0#80]
ICD-10 code R82.4 (Acetonuria) maps to two standard concepts in the `measurement` domain. I think one might actually need to be in the `procedure` domain instead.

This SQL:

``` sql
SELECT *
FROM concept_relationship
INNER JOIN concept icd10
ON icd10.concept_id = concept_id_1
INNER JOIN concept standard
ON standard.concept_id = concept_id_2
WHERE icd10.concept_code = 'R82.4'
AND relationship_id = 'Maps to';
```

finds these valid standard concepts: 4042243 (Urine ketone test) and 441968 (Acetonuria).

**Comment by cgreich** [comment]
@schuemie:

Yeah. That's a problem we don't quite know how to fix, yet. The problem is the following: Acetonuria, just like Anuria, Oliguria, Proteinuria are used both as Conditions as well as findings of a Measurement. SNOMED sometimes has them both, and sometimes only one of them. I already submitted that to them, and they accepted the change, but in the mean time we will have for some of them a dual mapping and for others a singular.

**Comment by ericaVoss** [comment]
Still looking like this is the same in 20161107.

**Comment by ericaVoss** [comment]
Still an issue in 20161218

**Comment by OlenaKalashnikova** [comment]
@cgreich 
As I understand if in a patient data we have R82.4, it means that doctor already took a look on the urinalysis results, compared it to the clinical signs and then put this R82.4 (Acetonuria) diagnosis. So it's a condition.

On the other hand if we have only urinalysis data, we'll get something like a LOINC code with results, and in this case it should go to measurement.

This way such ICD10CM concepts should be mapped **only** to findings not a lab tests.
Please let me know your thoughts.

**Comment by cgreich** [comment]
@OlenaKalashnikova: That's correct. The ICD10CM Acetonuria is a diagnosis, not a measurement in the urine.

**Comment by ericaVoss** [comment]
Just noting this is still the same in 20170920

**Comment by ericaVoss** [comment]
Just noting this seems to be the same in 20171201

**Comment by cgreich** [comment]
@dimshitc: You fixed the domain rules, didn't you? Did you also fix the SNOMED peaks?

**Comment by ericaVoss** [comment]
@schuemie this looks like it is fixed now.  Could you close if you agree?  

```sql
SELECT *
FROM concept_relationship
INNER JOIN concept icd10
ON icd10.concept_id = concept_id_1
INNER JOIN concept standard
ON standard.concept_id = concept_id_2
WHERE icd10.concept_code = 'R82.4'
AND relationship_id = 'Maps to';
```

Maps to:
441968	Acetonuria
4042243	Urine ketone test
